config/initializers/sidekiq.rb

```diff
@@ -18,7 +18,7 @@
   config.super_fetch! if defined?(Sidekiq::Pro)
 
   config.on(:startup) do
-    Sidekiq.schedule = YAML.load_file(File.expand_path('../sidekiq_scheduler.yml', __dir__))
+    Sidekiq.schedule = YAML.safe_load(ERB.new(File.read(File.expand_path('../sidekiq_scheduler.yml', __dir__))).result)
```

Comment:
Was this changed for development/testing purposes?

Reply:
I don't understand your question